collector-exoplayer/src/main/java/com/bitmovin/analytics/exoplayer/ExoPlayerAdapter.java

```diff
@@ -169,7 +169,7 @@ public void onPlayerStateChanged(boolean playWhenReady, int playbackState) {
                 }
                 break;
             case Player.STATE_BUFFERING:
-                if (this.stateMachine.getCurrentState() != PlayerState.SEEKING && this.stateMachine.getElapsedTimeFirstReady() != 0) {
+                if (this.stateMachine.getElapsedTimeFirstReady() != 0) {
```

Comment:
Are we sure we can just remove that? Otherwise the PR looks good to me

Reply:
i was also thinking that this was there on purpose
Testet it very well to understand it, but removed it in the end
Otherwise there are no rebuffering events after seeking
This might come from bitmovin player but here it works a little bit different

Bitmovin player
After seek has finished the player automatically transitions back to play
if no content ready -> rebuffering

Exo
Here we don't transition to play 
so we have a direct transition from seek to rebuffering

Reply:
If you tested it, then we're good to go 👍